crates/bitwarden-wasm-internal/src/platform/mod.rs

```diff
@@ -1,10 +1,28 @@
 use bitwarden_core::Client;
 use bitwarden_vault::{Cipher, Folder};
-use wasm_bindgen::prelude::wasm_bindgen;
+use tsify::{serde_wasm_bindgen, Tsify};
+use wasm_bindgen::prelude::*;
 
 mod repository;
 pub mod token_provider;
 
+#[derive(serde::Serialize, serde::Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+pub struct FlagsInput {
```

Comment:
Rather than duplicate the Flags struct, we should tag the Flags struct in bitwarden_core with:
```
#[cfg_attr(
    feature = "wasm",
    derive(tsify::Tsify),
    tsify(into_wasm_abi, from_wasm_abi)
)]
```

Otherwise we'll have to ensure we keep them both in sync, which is error prone.